### PR TITLE
[15.0][MIG] website_sale_product_detail_attribute_value_image: Migration to version 15.0

### PR DESCRIPTION
**Pull request by willytal**
cc @Tecnativa TT37334

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested. Thanks!

please @CarlosRoca13 review as well

File: website_sale_product_detail_attribute_value_image/models/product_attribute_value.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    website_product_detail_image = fields.Binary(
+        string="Website detail image",
+        attachment=True,
+        help="Image of the attribute value for shop online product detail.",
+    )
+    website_product_detail_image_published = fields.Boolean(
+        string="Publish detail image in website",
+        help="Display attribute value image in shop online product detail",
+    )
+    website_name = fields.Char(string="website_name", translate=True)
```
Comment (CarlosRoca13) [inline website_sale_product_detail_attribute_value_image/models/product_attribute_value.py]:
```suggestion
    website_name = fields.Char(translate=True)
```

**Review by CarlosRoca13 — COMMENTED** [review]
Minor comment

File: website_sale_product_detail_attribute_value_image/models/product_attribute_value.py (hunk 1 above)
Comment (willytal) [reply website_sale_product_detail_attribute_value_image/models/product_attribute_value.py]: @CarlosRoca13 done!

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge nobump
/ocabot migration website_sale_product_detail_attribute_value_image

**Pull request merged**